README.md

```diff
@@ -1 +1,1186 @@
+To view a quick guide to working with HarperDB and understand the basic concepts please explore our [Getting Started Example](https://examples.harperdb.io/)_
+
+_A typical flow for working with HarperDB is the following:_
+* Create a schema. Schemas are the starting point for a data model in HarperDB. Schemas are essentially a namespace that tables reside under.  
+    A schema is created by calling the HarperDBCreateSchema service
+* Create a table.  Tables reside under schemas and hold each object / row of data.  Create a table by calling the HarperDBCreateTable service
+* Insert Data.  HarperDB can insert both structured and unstructured data.  You can insert data via NoSQL with the HarperDBInsert Service and/or perform SQL INSERT with the HarperDBSQL Service.
+* Update Data. HarperDB can update both structured and unstructured data.  You can update data via NoSQL with the HarperDBUpdate Service and/or perform SQL UPDATE with the HarperDBSQL Service.
+* Query data by Hash (Primary Key). You can retrieve your data via a NoSQL search by hash function that returns data based on their hash value.  Search By Hash with the HarperDBSearchByHash service.
+* Query data by value.  Leveraging HarperDB's fully indexed data model you can search by value on any attribute in your data model.  Search by Value with the HarperDBSearchByVale service.
+* Query with SQL.  HarperDB has a robust SQL engine that allows you to perform full CRUD via SQL as well as perform aggregates, joins, conditions, etc... with the HarperDBSQL service.
+
+## Assets
+
+###
+
+- `HarperDBCreateSchema` - Creates a new schema in HarperDB
+- `HarperDBCreateTable` - Creates a new table in an existing schema in HarperDB
+- `HarperDBCSVDataLoad` - Loads CSV formatted data into a HarperDB table
+- `HarperDBDelete` - Deletes data by hash (primary key) in HarperDB
+- `HarperDBDescribeAll` - Returns the schema/table/attribute meta data for all schemas
+- `HarperDBDescribeSchema` - Returns the schema/table/attribute meta data for a schema
+- `HarperDBExecuteOperation` - Execute a raw HarperDB call
+- `HarperDBInsert` - Inserts JSON data into an existing table in HarperDB
+- `HarperDBSearchByHash` - Returns an object array based on a search by hash (primary key)
+- `HarperDBSearchByValue` - Returns an object array based on a search by value on an attribute
+- `HarperDBSQL` - Run SQL (INSERT / UPDATE / DELETE / SELECT)
+- `HarperDBUpdate` - Updates data in an existing table in HarperDB
+
+### Libraries
+
+- `HarperDB` - Library that executes core operations against HarperDB
+- `HarperDBConfiguration` - Configuration settings to connect to HarperDB
+
+## API
+
+_Recommend using JSDoc for documenting your JavaScript code, and using jsdoc2md npm tool (https://www.npmjs.com/package/jsdoc-to-markdown) to generate markdown_
+
+```
+npm install -g jsdoc-to-markdown
+jsdoc2md code/*/*/*.js >> Readme.md
+```
+## Classes
```

Comment:
Use 
   "jsdoc2md code/libraries/*/*.js --heading-depth=3 > jsdoc.md"
   
So that each heading is not big.